From 5b3935fa43c91e4aea62b65fd0bdf4ac7771e841 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Florian=20M=C3=BCllner?= <fmuellner@gnome.org>
Date: Tue, 20 Aug 2019 02:31:52 +0200
Subject: [PATCH 113/152] cleanup: Fix up "special" comments

Our coding style asks for a space after the comment start, which
some of our ASCII-artsy comments violate. Adjust them to fit the
rule, or remove them altogether.

https://gitlab.gnome.org/GNOME/gnome-shell/merge_requests/731
---
 js/misc/modemManager.js            | 12 ++++++------
 js/perf/hwtest.js                  |  8 +++-----
 js/ui/components/autorunManager.js |  2 --
 3 files changed, 9 insertions(+), 13 deletions(-)

diff --git a/js/misc/modemManager.js b/js/misc/modemManager.js
index f7240b67c..673db60f8 100644
--- a/js/misc/modemManager.js
+++ b/js/misc/modemManager.js
@@ -84,9 +84,9 @@ function _findProviderForSid(sid) {
 }
 
 
-//------------------------------------------------------------------------------
-// Support for the old ModemManager interface (MM < 0.7)
-//------------------------------------------------------------------------------
+// ----------------------------------------------------- //
+// Support for the old ModemManager interface (MM < 0.7) //
+// ----------------------------------------------------- //
 
 
 // The following are not the complete interfaces, just the methods we need
@@ -182,9 +182,9 @@ var ModemCdma = class {
 Signals.addSignalMethods(ModemCdma.prototype);
 
 
-//------------------------------------------------------------------------------
-// Support for the new ModemManager1 interface (MM >= 0.7)
-//------------------------------------------------------------------------------
+// ------------------------------------------------------- //
+// Support for the new ModemManager1 interface (MM >= 0.7) //
+// ------------------------------------------------------- //
 
 const BroadbandModemInterface = loadInterfaceXML('org.freedesktop.ModemManager1.Modem');
 const BroadbandModemProxy = Gio.DBusProxy.makeProxyWrapper(BroadbandModemInterface);
diff --git a/js/perf/hwtest.js b/js/perf/hwtest.js
index fa2456323..be7392984 100644
--- a/js/perf/hwtest.js
+++ b/js/perf/hwtest.js
@@ -137,9 +137,9 @@ function *run() {
     Main.overview.hide();
     yield Scripting.waitLeisure();
 
-    ////////////////////////////////////////
-    // Tests of redraw speed
-    ////////////////////////////////////////
+    // --------------------- //
+    // Tests of redraw speed //
+    // --------------------- //
 
     global.frame_timestamps = true;
     global.frame_finish_timestamp = true;
@@ -186,8 +186,6 @@ function *run() {
 
     yield Scripting.sleep(1000);
 
-    ////////////////////////////////////////
-
     let appSys = Shell.AppSystem.get_default();
     let app = appSys.lookup_app('org.gnome.gedit.desktop');
 
diff --git a/js/ui/components/autorunManager.js b/js/ui/components/autorunManager.js
index cb0c6b6a3..751c45a89 100644
--- a/js/ui/components/autorunManager.js
+++ b/js/ui/components/autorunManager.js
@@ -72,8 +72,6 @@ function startAppForMount(app, mount) {
     return retval;
 }
 
-/******************************************/
-
 const HotplugSnifferIface = loadInterfaceXML('org.gnome.Shell.HotplugSniffer');
 const HotplugSnifferProxy = Gio.DBusProxy.makeProxyWrapper(HotplugSnifferIface);
 function HotplugSniffer() {
-- 
2.23.0

